guides/cursor.mdx

```diff
@@ -0,0 +1,394 @@
+---
+title: "Cursor"
+description: "Configure Cursor to be your writing assistant"
+icon: "box"
+---
+
+Transform Cursor into a documentation expert that knows your components, style guide, and best practices.
+
+## Using Cursor with Mintlify
+
+Cursor rules provide persistent context about your documentation, ensuring more consistent suggestions that fit your standards and style.
+
+* **Project rules** are stored in your documentation repository and shared with your team.
+* **User rules** apply to your personal Cursor environment.
+
+We recommend creating project rules for your docs so that all contributors have access to the same rules.
+
+Create rules files in the `.cursor/rules` directory of your docs repo. See the [Cursor Rules documentation](https://docs.cursor.com/context/rules) for complete setup instructions.
+
+## Example project rules
+
+This rule provides Cursor with knowledge of frequently used Mintlify components and technical writing best practices. Copy and paste this rule into a `.mdc` file in your `.cursor/rules` directory or adjust it for your documentation.
+
+Customize this rule for your project:
+- **Writing standards**: Update language guidelines to match your style guide
+- **Component patterns**: Add project-specific components or modify existing examples
+- **Code examples**: Replace generic examples with real API calls and responses for your product
+- **Style and tone preferences**: Adjust terminology, formatting, and other rules
+
+````
```

Comment:
The transition from the description above to this block is a little bit jarring. I know it's stated as `## Example project rules`, but I think it might work better to have one more subheading before the block, since it goes

`Example Project Rules`
This rule...

A list

The aforementioned rule

Reply:
Great point ✨ . Resolved with the latest commit.